src/intl/en.json

```diff
@@ -71,11 +71,25 @@
     "tabTitle": "When and how can I come or participate?",
     "title": "When and how can I come or participate?",
     "subtitle": "To set a volunteering day or a personal retreat, or to come for a visit, please contact Einat:",
+    "sub1":"Groups visits: ",
+    "sub2":"Private Visits: ",
+    "sub3":"Consultat: " ,
+    "sub4":"Learning Days: ",
+    "sub5":"Retreats: ",
+    "sub6":"Comminity Gatherings: ",
+    "sub7":"Howtos: ",
+    "sub8":"Buy: ",  
     "text": [
       "Our ecological apartment is a real model of urban, ecological living. You can come to short tours by setting the time with us in advance. It costs NIS25 per person. You can participate in our work and study through volunteering days at times suitable for you and us.",
       "You can join a scheduled event such as “vinegar day”, “bokashi day” or “sewing day”, full moon gatherings or seasonal community gatherings. We send weekly updates (in Hebrew) in our  newsletters as well as on our Whatsapp group and Facebook pageWe also offer the option to design your own personal retreat. People come here to study, experience practical ecological living, meet companions, rest and heal.",
+      "To be translated soon...",
+      "To be translated soon...",
+      "To be translated soon...",
+      "You can also join us for a longer period of life and learning, be exposed to ecological practice, meet companions, rest and heal, whether in our seasonal retreats or by personal coordination.",
+      "To be translated soon...",
+      "To be translated soon...",
       "Our crazy little shop is open! You can set a time and come buy vinegar, bokashi, soap or other zero-waste products. Please visit the <shop>online shop</shop> that is gradually being enriched with more products.",
-      "You can also join us for a longer period of life and learning, be exposed to ecological practice, meet companions, rest and heal, whether in our seasonal retreats or by personal coordination."
+      "To be translated soon..."
```

Comment:
The "schedule" section must also be updated - it's broken in En at the moment.

Reply:
ok, already notified Alon to be on that